packages/smooth_app/lib/pages/user_management/sign_up_page.dart

```diff
@@ -131,9 +132,17 @@ class _SignUpPageState extends State<SignUpPage> {
                 }
                 return null;
               },
+              onSubmit: (String? value) {
+                // Checks if confirm password is equal to password if not then
+                // move the focus to the confirm password field
+                if (_password2Controller.text != value) {
+                  FocusScope.of(context).requestFocus(confirmPassword);
+                }
```

Comment:
So two things

1. I could imagine it beeing a bit confusing if sometimes you jump to the second password and another time two steps further. Its certainly great when you know it, but when not not
2. Please add a comment here and where you create the Focus node that there was a bug and thats why we have this hacky way for changing the focus only for one field

Reply:
@M123-dev sure I'll do the points marked out.

Reply:
Any update @CyberWake